demo/meeting/src/components/MediaStatsList/MetricItem.tsx

```diff
@@ -0,0 +1,34 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import React from 'react';
+import { StyledItemMetricName, StyledItemMetricValue } from './Styled';
+
+export interface MetricItemProps {
+  metricName: string;
+  metricValues: string[];
+}
+
+export const MetricItem: React.FC<MetricItemProps> = ({
+  metricName,
+  metricValues,
+}) => {
+  const showMetricItem = metricValues[0] !== '';
+  return (
+    <>
+      {showMetricItem && (
+        <>
+          <StyledItemMetricName>{metricName}</StyledItemMetricName>
+          {metricValues &&
```

Comment:
`metricValues[0]` is checked for `showMetricItem`, so no much use of `metricValues &&` here right? May be checked at the upstart itself.

Reply:
That is true. We can remove the `metricValues && `. Good catch.